### PR TITLE
[RAV][nextjs] Ensure component works when default personalization variant is hidden

### PR DESCRIPTION
**Pull request by art-alexeyenko**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)

## Description / Motivation
Personalized components could render an error frame when default variant is hidden. This PR fixes it.

## Testing Details
- [x] Unit Test Added
- [x] Manual Test/Other  - should work in normal mode, Pages, Experience Editor

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx [hunk 1]
```diff
@@ -211,6 +211,8 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
 
         if (componentRendering.componentName === HIDDEN_RENDERING_NAME) {
           component = hiddenRenderingComponent ?? HiddenRendering;
+        } else if (!componentRendering.componentName) {
+          component = () => <></>;
```
Comment (illiakovalenko) [inline packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx:215]: It's possible to render `null` value, did you try to set `null`?

**Review by illiakovalenko — APPROVED** [review]
Looks good to me! See one suggestion below

File: packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx (hunk 1 above)
Comment (art-alexeyenko) [reply packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx:215]: I think either one should work, this approach is a better fit for the code flow we have now (there's a null check below that actually causes the issue).

Comment (art-alexeyenko) [reply packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx:215]: Also it helps differentiate between an "empty" component (all good - means it's hidden) and "null" component (throws error, causes error frame to render).

**Pull request merged**